src/strategies/tests/poap-holder-of.test.ts

```diff
@@ -22,7 +22,7 @@ describe('poap holder of', () => {
     expect(strategyProvider.strategies[0][0].enrichedInformations.symbol).toBeDefined();
     expect(strategyProvider.strategies[0][0].enrichedInformations.symbol).toBeDefined();
   });
-  test('should return false if no addresses attended POAP', async () => {
+  xtest('should return false if no addresses attended POAP', async () => {
```

Comment:
test.skip?

Reply:
Yes, xtest is a test skip.
And it is the same problem for POAP like Rinkeby, as you told me